### PR TITLE
CAT-1762 DefaultDroneProject refactored

### PR DESCRIPTION
**Pull request by MarkusHobisch**
https://jira.catrob.at/browse/CAT-1762

### CODE REVIEW

**Comment by robertpainsi** [comment]
This [PR@8b8b62c](https://github.com/MarkusHobisch/Catroid/commit/8b8b62c84ded228ed01c60129b9f0dcb80b43835) is not formatted even though the format status says so. I'm working on a fix...

**Comment by MarkusHobisch** [comment]
OK well done. Please fix it also in the SourceTests.

**Comment by robertpainsi** [comment]
The formatter is back in action! :smiley_cat:

> Please fix it also in the SourceTests.

Definitely, [CAT-1549](https://jira.catrob.at/browse/CAT-1549)

File: catroid/src/org/catrobat/catroid/common/DefaultProjectHandler.java [hunk 1]
```diff
@@ -122,12 +131,18 @@ public static Project createAndSaveDefaultDroneProject(String projectName, Conte
 
 		// Background sprite
 		backgroundSprite.getLookDataList().add(backgroundLookData);
+		backgroundSprite.getLookDataList().add(droneVideoLookData);
 		Script backgroundStartScript = new StartScript();
 
 		SetLookBrick setLookBrick = new SetLookBrick();
-		setLookBrick.setLook(backgroundLookData);
-		backgroundStartScript.addBrick(setLookBrick);
+		TurnLeftBrick turnLeftBrick = new TurnLeftBrick(90f);
```
Comment (fwinkelbauer) [inline catroid/src/org/catrobat/catroid/common/DefaultProjectHandler.java]: should 90f and 200.0 be defined as private static finals?

**Comment by thmq** [comment]
this will be re-implemented by new drone team.

**Pull request closed without merging**